### ISSUE TITLE
Switch language by itself--feature or bug?

### ISSUE THREAD

**Issue by flexid2000** [zen-tools/gxkb#17]
Installed on Debian Stretch (gxkb (0.7.8-1)), the programme switches by itself from one language to the next as soon as a new window is opened. That enables different windows opened at the same time, each using a different keycode. Is it possible to stop this behaviour?
My `gxkb.cfg`:
```
[xkb config]
group_policy=2
default_group=0
never_modify_config=false
model=pc105
layouts=it,de
variants=
toggle_option=
compose_key_position=
```

**Comment by flexid2000** [comment]
The order of the values in `layouts` in the `gxkb.cfg` file seems to be relevant. The programme uses the first value as the default keycode. After switching a new window is opened defaulting to the first value. Changing the order of the values stops the automatic switching.

**Comment by zen-tools** [comment]
Hi! It seems like you have running two processes to control keyboard layout. Try to disable the inheritance of the layouts from parent window and disable splitting layouts between windows in your DE settings.

**Comment by flexid2000** [comment]
You're right, the question is not completely solved. But since I'm not so much of an expert of the openbox window manager I wouldn't know how to disable child windows inheriting their parents' settings or how to stop splitting layouts between windows. By changing the order of the values in the layout item at least I found a feasible way to control the default layout.

**Comment by zen-tools** [comment]
Can you show the output of the command `ps aux` in terminal? That should help me in the investigation.

**Comment by flexid2000** [comment]
Here it is:
https://pastebin.com/8v54SuSc

**Comment by zen-tools** [comment]
I believe the bug was fixed in v0.8.2. Feel free to reopen if the issue appears again.
